dev-python/pyenchant/pyenchant-3.2.0.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Python bindings for the Enchant spellchecker"
+DESCRIPTION="This package provides a set of Python language \
+bindings for the Enchant spellchecking library. For more \
+information, visit the project website."
+HOMEPAGE="http://pyenchant.github.io/pyenchant/"
+COPYRIGHT="2020 Dimitri Merejkowsky"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/pyenchant/pyenchant/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="3195bfc5cfbdeb8a60f9484390c2c982e59dd220d6f9a5f8418ba67551ead01a"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libenchant_2$secondaryArchSuffix
```

Comment:
As "lib:libenchant_2$secondaryArchSuffix" isn't "any" architecture, I guess this means this can't be "any" architecture, see failed build: https://build.haiku-os.org/buildmaster/master/x86_gcc2/?buildrunDir=1159&viewMode=expanded

Reply:
Yes, that makes sense.

Reply:
I just testet it with the first requirement and I had used the suggestion from korli without testing it again. Sorry for that.

Reply:
to be honest, I don't expect everything to work out of the box on x86, because python3 is only x86 secondary arch. It means that it should load x86 secondary arch libraries.
@Begasus  Maybe we could check the effectiveTargetArchitecture, and add a suffix in this case as a workaround.

Reply:
Not a big fan for these "mixed" architectures with the python recipes, but an option to check yes

Reply:
I pushed a small change to: https://github.com/haikuports/haikuports/pull/5675
This fixes the build for me, but still seems to fail on the buildbot :(